src/cli/src/cluster/check.rs

```diff
@@ -3,22 +3,15 @@ use std::io::ErrorKind;
 use std::str::FromStr;
 use std::net::{IpAddr};
 use std::process::Command;
-use std::time::Duration;
 
 use semver::Version;
 use k8_client::K8Config;
-use fluvio_future::timer::sleep;
-use k8_client::ClientError;
-use k8_client::load_and_share;
-use k8_obj_core::service::ServiceSpec;
-use k8_obj_metadata::InputObjectMeta;
-use k8_client::ClientError as K8ClientError;
 use k8_config::KubeContext;
 use url::{Url};
 use structopt::StructOpt;
 use colored::*;
 use serde_json::{Value};
-use fluvio_cluster::ClusterInstaller;
+use fluvio_cluster::{ClusterInstaller, check_load_balancer_status};
```

Comment:
This is causing me to think though some possible steps for a Check refactoring. I think that all of the remaining check logic in the CLI crate should be moved to the cluster crate into the check module. I don't like this standalone `check_load_balancer_status` function being made public. I think we should have a one-stop-shop for checking purposes, something like a `ClusterChecker` to complement the `ClusterInstaller`. Can we rename this into `_check_load_balancer_status` and mark it with a `#[doc(hidden)]` attribute so that it is not considered a permanent part of our public API? I'm fine with keeping it for now if we do that, but later I'd like to clean things up and remove that export.

Reply:
See the `_install_sys` function in `fluvio_cluster::install` to see an example of the hidden function

Reply:
there is issue for moving all cluster related logic into cluster module:  https://github.com/infinyon/fluvio/issues/335

Reply:
I agree, we should make as hidden for now